pkg/cloudprovider/azure/object_storage_adapter.go

```diff
@@ -93,6 +93,10 @@ func (op *objectStorageAdapter) ListCommonPrefixes(bucket string, delimiter stri
 		Delimiter: delimiter,
 	}
 
+	if prefix != "" {
```

Comment:
What happens if you pass this even when it's `""`?

Reply:
same as above